src/me/ryanhamshire/PopulationDensity/Messages.java

```diff
@@ -64,5 +64,6 @@
     RegionNamesOnlyLettersAndNumbers,
     RegionAlreadyNamed,
     HopperLimitReached,
-    OutsideWorldBorder
+    OutsideWorldBorder,
+    Wilderness,
```

Comment:
Syntax error, won't compile

Reply:
Oh weird, that's not an error in Java 8? Interesting.

Reply:
Still though, if you could remove the trailing comma that'd be appreciated.

Reply:
Sure, I've removed it. The reason I included the trailing comma is because it makes version control changes cleaner when adding new elements to lists, but it's a style preference anyway, and this is your repo! Thanks for the quick response.